parser-combinators.cabal

```diff
@@ -31,6 +31,9 @@ library
   if !impl(ghc >= 8.0)
     build-depends:    semigroups == 0.18.*
 
+  other-extensions:   CPP
```

Comment:
Can you remind me what's the reason for having `other-extensions`?

Reply:
I believe it'll make Cabal reject the package immediately if the implementation doesn't support those extensions. I doubt it's of much practical significance in this case, but it seems to be common practice to include it.

Reply:
I think that nothing prevents this to go out of sync with reality. I'd certainly forget to update the list. Actually, I think it's pretty feasible for Cabal to infer the list of used extensions from source files and use that instead of explicit list provided by user. I remember there is some stuff in GHC for scanning enabled extensions and turning them into dyn flags.

Anyway, I think I'll drop this one, as it seems to be indeed of little practical significance.